### PR TITLE
docs: Describe TeamForCapella commit and commit description options

### PR DESCRIPTION
**Pull request by MoritzWeber0**
Described how to activate commit messages / descriptions, how to enter a commit message and describe the commit history.

### CODE REVIEW

File: docs/mkdocs.yml [hunk 1]
```diff
@@ -146,6 +147,6 @@ markdown_extensions:
 extra:
   generator: false
 
-dev_addr: 127.0.0.1:8081
+dev_addr: 127.0.0.1:10002
```
Comment (romeonicholas) [inline docs/mkdocs.yml:150]: Was this intentional? Will just require updating the development/docs.md instructions

File: docs/docs/user/tools/capella/teamforcapella/commit-messages/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ -->
+
+# Work with TeamForCapella commits and commit messages
+
+!!! Info
+
+    Please [request a persistent Capella session](../../../../sessions/request.md) before you continue
+    with the instructions here.
+
+## Activate commit messages
```
Comment (romeonicholas) [inline docs/docs/user/tools/capella/teamforcapella/commit-messages/index.md:13]: I might rename this to "Enable commit message prompt" (I think "activate" might be misunderstood here as activating commits on the model itself, not the user-specific experience)

**Review by romeonicholas — CHANGES_REQUESTED** [review]
Very clear instructions, good screenshots 👍

File: docs/mkdocs.yml (hunk 1 above)
Comment (MoritzWeber0) [reply docs/mkdocs.yml:150]: No, it wasn't intentional. I just a had a few port conflicts in my local environment. At some point, we have to align the ports between the projects. I'll revert his change.

File: docs/docs/user/tools/capella/teamforcapella/commit-messages/index.md (hunk 2 above)
Comment (MoritzWeber0) [reply docs/docs/user/tools/capella/teamforcapella/commit-messages/index.md]: Done

**Pull request merged**